src/content/configuration/output.mdx

```diff
@@ -1187,49 +1187,32 @@ T> Using some kind of indentation makes bundles look prettier, but will cause is
 
 T> Typically you don't need to change `output.sourcePrefix`.
 
-## `output.strictModuleExceptionHandling`
```

Comment:
It's deprecated but still available to use. So let's add a warning and keep it at the moment until it's removed from webpack.